modules/porous_flow/src/materials/PorousFlowVolumetricStrain.C

```diff
@@ -64,6 +65,12 @@ PorousFlowVolumetricStrain::PorousFlowVolumetricStrain(const InputParameters & p
 }
 
 void
+PorousFlowVolumetricStrain::initQpStatefulProperties()
+{
+  _vol_total_strain_qp[_qp] = 0.0;
```

Comment:
I need this initQpStatefulProperties, otherwise MOOSE complains: 

Material "vol_strain" provides one or more stateful properties but initQpStatefulProperties() was not overridden in the derived class.